docs/MapColonies/Raster/services/export.mdx

```diff
@@ -31,16 +31,11 @@ You can create export tasks via the [Exporter-Trigger service API](/docs/MapColo
 
 You can see the full OpenAPI spec [here](<RASTER-EXPORT-SERVICE_URL>/docs/api)
 
-The API consists of several methods:
-1. create (deprecated) - you can create a new export task, if the requested task params were previously created they are returned in the response, else via provided webhook
-:::caution
-The create API is in use of GetMap. This use is now deprecated. Use `create/roi` instead.
-:::
-2. create/roi - you can create a new export task from ROI (region of intreset - GoeJson FeatureCollection), if the requested task params were previously created they are returned in the response, else via provided webhook
-3. taskStatus/{jobId} - Get status (and percentage) by the id that is returned from the create method
-4. storage - returns free and total existing size on export service storage
+The API currently consists of one method:
+1. POST /export-tasks - Create a new export task, if the requested task params were previously created they are returned in the response, else via provided webhook
 
 ## Files structure
+Under "artifacts" section in the response you will find the following artifacts
 
 #### GPKG
 SQLite DB file that contains all the necessary tiles
```

Comment:
Please remove `[here]` in line 47 and replace with `[terms]`.
Please fix link in line 48 and put it on `[Tiling scheme]`.

Reply:
fixed